feature-libs/product/bulk-pricing/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './bulk-pricing.module';
```

Comment:
I'm not convinced here yet, I think we might want to create separate entry points for core, occ, and components insstead of grouping them here. Will discuss it and let you know.

Reply:
Separate entry points added